lib/segment/src/vector_storage/dense/memmap_dense_vector_storage.rs

```diff
@@ -191,7 +191,7 @@ impl<T: PrimitiveVectorElement> VectorStorage for MemmapDenseVectorStorage<T> {
         // Extend vectors file, write other vectors into it
```

Comment:
Here is the important bit. We don't use the provided point offset, but regenerate our own offset sequence.